app/javascript/packs/miq-redux-common.ts

```diff
@@ -0,0 +1,24 @@
+import { IModule } from 'angular';
```

Comment:
should we extract angular specifics into a different pack / file? just wondering if we move away from angular, ideally redux core implementation should not change

Reply:
Following files depend on Angular specifics (1.x module API):
* `ng-redux` based Redux store configuration: `miq-redux/store.ts`
* pack entry point: `packs/miq-redux-common.ts`
* test helper (to allow testing specific module exports): `miq-redux/test-helper.ts`

If we'd decide to move away from Angular, we'd have to change store creation, entry point code as well as tests (which currently use `inject` to accept Angular managed objects).

However, I could extract all Angular stuff as shown above into a single file, say `miq-redux/angular-config.ts`.